### DISCUSSION TITLE
hyperlinks to tagged sections in other qmd

### DISCUSSION THREAD

**Discussion by irisoren-sg** [best-practice-and-impact/aqua_book_revision#174] [General]
@zarbspace Do you know how to hyperlink to sections from one qmd to another. My attempts have not worked See [definitions_and_key_concepts line 175 ](https://github.com/best-practice-and-impact/aqua_book_revision/blob/733a3b3329e3ae4182bca5f86e1970f3cacdc20d/definitions_and_key_concepts.qmd#L175)and [engagement_and_scoping.qmd](https://github.com/best-practice-and-impact/aqua_book_revision/blob/733a3b3329e3ae4182bca5f86e1970f3cacdc20d/engagement_and_scoping.qmd#L54)

**Comment by kateod-ukhsa** [comment]
@irisoren-sg 

The hyperlink should work if you remove the forward slash `/` :)

**Before**
https://github.com/best-practice-and-impact/aqua_book_revision/blob/4df227ba471b8bfe4593866e97706fb95a2e516d/definitions_and_key_concepts.qmd#L172

**After**
```suggestion
See [Roles and Responsibilities](analytical_lifecycle.qmd#roles_and_responsibilities) for details.
```
